assets/erc-7730/erc7730-v1.schema.json

```diff
@@ -66,19 +66,6 @@
                     "description": "The contract binding context is a set constraints that are used to bind the ERC7730 file to a specific smart contract.",
 
                     "properties": {
-                        "abi": {
```

Comment:
Can we leave the ABI key and mark it as deprecated? To avoid breaking a lot of existing files?

Reply:
added a short mention